pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -34,6 +40,12 @@ type DirectVolumeMigrationSpec struct {
 	SrcMigClusterRef  *kapi.ObjectReference `json:"srcMigClusterRef,omitempty"`
 	DestMigClusterRef *kapi.ObjectReference `json:"destMigClusterRef,omitempty"`
 
+	// BackOffLimit retry limit on Rsync pods
+	BackOffLimit int `json:"backOffLimit,omitempty"`
+
+	// RsyncPodGarbageCollection whether to clean up failed pods during retry
+	RsyncPodGarbageCollection bool `json:"rsyncPodGarbageCollection,omitempty"`
```

Comment:
I wonder if this needs to be exposed in the API?

Reply:
@alaypatel07 Keeping in the api in an effort to reduce the dependencies (thinking about state only transfers), open to moving this around or moving to MigrationController spec instead

Reply:
@pranavgaikwad is there any value in allowing users to configure this? Can we just decide for the user that we will garbage collect pods by default or may be have a global setting for this at migrationcontroller CR level? It feels to me this is increasing the fields in API which we don't necessarily absolutely need.

Reply:
@alaypatel07 agreed. I think we should remove the API field and move it to MigrationController. Removed